cmd/tailscale/cli/share.go

```diff
@@ -108,10 +109,10 @@ func runShareRename(ctx context.Context, args []string) error {
 	return err
 }
 
-// runShareList is the entry point for the "tailscale share list" command.
+// runShareList is the entry point for the "tailscale drive list" command.
 func runShareList(ctx context.Context, args []string) error {
```

Comment:
kinda weird for "tailscale drive list" to have no "share" in it but this func to have "share" in it (runShareList).

Reply:
Fixed.